Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -38,7 +38,7 @@
 #import "RemoteScrollingCoordinatorProxyIOS.h"
 #import "ScrollingTreeScrollingNodeDelegateIOS.h"
 #import "TapHandlingResult.h"
-#import "UIKitSPI.h"
```

Comment:
I imagine this is getting pulled in from another header, but there is other SPI usage in this file and I think we should keep the import explicit.

Same comment applies to the other removals (unless there's one where we aren't actually using any other SPI).